src/Attributes/Attribute.php

```diff
@@ -83,6 +87,14 @@ public static function tryFromValue($value): ?self
             return new self($value, 'double');
         }
 
+        if ($value instanceof SerializableInterface) {
+            try {
+                return new self(JSON::encode($value->toSentry()), 'string');
+            } catch (\Throwable $e) {
+                // Ignore the exception and continue trying other methods
+            }
+        }
```

Comment:
Because we have it, I prefered this over the __toString or just raw JSON encode.

For safety we do ignore any errors to prevent crashing because serializing fails.